tests/test_admin.py

```diff
@@ -576,3 +577,23 @@ def test_change_view(self):
         self.assertEqual(self.category.name, 'test category')
         self.category.set_current_language('de')
         self.assertEqual(self.category.name, 'Alias Kategorie')
+
+
+class AliasesManagerTestCase(BaseAliasPluginTestCase):
+
+    def test_aliases_admin_entry_is_hidden(self):
```

Comment:
Could also add a check to ensure that the admin for alias can still be accessed via url.

Reply:
Done